### ISSUE TITLE
Reading PSSE raw files or CIM files

### ISSUE THREAD

**Issue by linar-jether** [PyPSA/PyPSA#24]
Any simple way to import a cim model or psse raw file?

**Comment by nworbmot** [comment]
No. You'll have to write your own importer I'm afraid. The PyCIM project might be useful. I think the pandapower team https://github.com/lthurner/pandapower has these importers, but they may not be public.

**Comment by fneum** [comment]
Reopening this because it got relevant again.

Also: PyCIM is now CIMpy: https://github.com/sogno-platform/cimpy
